### PR TITLE
build: added IJ-2022.3, min to IC-2021.1, jdk 17, gradle 7.6, intellij-gradle 1.10.1 (#63)

### LINKED ISSUE #63: Add IJ 2022.3 verification
IJ 2022.3 is out

### PR DESCRIPTION
**Pull request by adietish**
fixes #63

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
   ]]>
     </change-notes>
 
-    <idea-version since-build="193.5233.102" />
+    <idea-version since-build="21.1" />
```
Comment (jeffmaury) [inline src/main/resources/META-INF/plugin.xml:29]: This value looks strange to me

File: .github/workflows/IJ.yml [hunk 2]
```diff
@@ -14,33 +14,37 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IC-2019.3, IC-2020.1, IC-2020.2, IC-2020.3, IC-2021.1, IC-2021.2, IC-2021.3, IC-2022.1, IC-2022.2]
+        IJ: [IC-2021.1, IC-2021.2, IC-2021.3, IC-2022.1, IC-2022.2, IC-2022.3]
```
[the remark this answers is not in the document]
Comment (adietish) [reply .github/workflows/IJ.yml]: all our plugin now require IC-2021.1 but I can go back to 2019.3

[the remark this answers is not in the document]
Comment (jeffmaury) [reply .github/workflows/IJ.yml]: https://github.com/redhat-developer/intellij-quarkus/blob/main/gradle.properties

[the remark this answers is not in the document]
Comment (adietish) [reply .github/workflows/IJ.yml]: the reason to bump the min IC version was the error that github actions now run into:
```
org.gradle.process.internal.ExecException: Process 'Gradle Test Executor 2' finished with non-zero exit value 1
	at org.gradle.process.internal.DefaultExecHandle$ExecResultImpl.assertNormalExitValue(DefaultExecHandle.java:415)
	at org.gradle.process.internal.worker.DefaultWorkerProcess.onProcessStop(DefaultWorkerProcess.java:145)
	at org.gradle.process.internal.worker.DefaultWorkerProcess.access$000(DefaultWorkerProcess.java:42)
	at org.gradle.process.internal.worker.DefaultWorkerProcess$1.executionFinished(DefaultWorkerProcess.java:98)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:43)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:245)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:157)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:141)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:37)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at jdk.proxy1/jdk.proxy1.$Proxy114.executionFinished(Unknown Source)
	at org.gradle.process.internal.DefaultExecHandle.setEndStateInfo(DefaultExecHandle.java:221)
	at org.gradle.process.internal.DefaultExecHandle.finished(DefaultExecHandle.java:355)
	at org.gradle.process.internal.ExecHandleRunner.completed(ExecHandleRunner.java:110)
	at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:84)
	at org.gradle.internal.operations.CurrentBuildOperationPreservingRunnable.run(CurrentBuildOperationPreservingRunnable.java:42)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Unrecognized VM option 'UseConcMarkSweepGC'
Error: Could not create the Java Virtual Machine.
```
This goes away when bumping target to `11` (was: 18) and bumping min version to `IC-2019.3`. 
I can try to bump to `IC-2020.1` and see if it fixes it.

[the remark this answers is not in the document]
Comment (adietish) [reply .github/workflows/IJ.yml]: using `IC-2020.1` the error still exists. Trying `IC-2020.3`

File: .github/workflows/IJ.yml [hunk 3]
```diff
@@ -14,33 +14,51 @@ jobs:
...
-    - name: Set up JDK 11
-      uses: actions/setup-java@v1
-      with:
-        java-version: 11
-    - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
-    - name: Build with Gradle
-      run: ./gradlew build -PideaVersion=${{ matrix.IJ }}
+      - uses: actions/checkout@v2
+      - name: Set up JDK 17
```
Comment (jeffmaury) [inline .github/workflows/IJ.yml:35]: I think this need to use ${{matrix.jvm }} in place of 17 not to confuse readers

**Review by jeffmaury — CHANGES_REQUESTED** [review]

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**